### PR TITLE
(DOCSP-3554): Documenting collation field in query bar

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
Some small refactoring done as part of this ticket as well.

Staged: https://docs-mongodbcom-staging.corp.mongodb.com/compass/jeffreyallen/DOCSP-3554/query-bar.html#query-bar-collation

### CODE REVIEW

File: source/query-bar.txt [hunk 1]
```diff
@@ -164,8 +184,37 @@ sort by ``name`` in ascending order.
...
+
+.. _query-bar-collation:
+
+Collation
+~~~~~~~~~
+
+:manual:`Collation </reference/collation/>` allows you to specify
+language-specific rules for string comparison, such as rules for
+lettercase and accent marks. Use this query bar option to specify
+collation for the current query.
```
Comment (rob-mongodb) [inline source/query-bar.txt]: The second sentence feels like filler to me.  Doesn't really add anything to the first sentence. Perhaps merge with the first sentence? "Use the Collation query bar option to ..."

File: source/query-bar.txt [hunk 2]
```diff
@@ -75,31 +82,44 @@ The Compass :guilabel:`Filter` supports using the :binary:`~bin.mongo`
...
 .. code-block:: javascript
 
    { "_id": ObjectId('59a87101f17fcbfbc9cd4374') }
 
+Query Bar Autocompletion
+````````````````````````
+
+The query bar autocompletes the current query based on the keys in
+your collection’s documents, including keys in embedded
+sub-documents.
```
Comment (rob-mongodb) [inline source/query-bar.txt]: I don't think we need a hyphen here.  Consider "subdocuments".

File: source/query-bar.txt (hunk 1 above)
Comment (jeff-allen-mongo) [reply source/query-bar.txt]: Agreed. Done.

**Comment by jeff-allen-mongo** [comment]
Hey @durran, would you mind taking a look at these updates and making sure we have everything covered?

**Comment by durran** [comment]
See it's already merged but LGTM. :)

**Pull request merged**